content/quickstart.md

```diff
@@ -7,29 +8,72 @@ tocopen: true
 
 # Prerequisites
 
-Under construction.
+1- OpenSeeFace tracking: webcam
 
-# Installation
+2- IFacialMocap tracking: iOS device with face ID or A12 (or newer) chip
+
+3- MeowFace tracking: android device that supports ARCore (most modern phones do)
 
-Under construction. Grab something from [downloads]({{< ref "downloads/">}}) :)
+4- 64-bit Operating system with x86 architecture
+
+# Installation
 
 ## Windows
 
-Under construction. Grab something from [downloads]({{< ref "downloads/">}}) :)
+1- download windows [zip file](https://github.com/virtual-puppet-project/vpuppr/releases)
+
+2- unzip files into a directory, (do not move files!)
+
+3- run the ``.exe`` on windows
+
+4- start the facetracker from within the app!
 
 ## Linux
 
 ### Flatpak
 
-Under construction.
+#### OpenSeeFace-GD (old)
+```
+    $ flatpak install flathub com.github.virtual_puppet_project.vpuppr
+```
+
+#### VPupPr (Alpha 2)
+```
+    $ flatpak remote-add --if-not-exists flathub-beta https://flathub.org/beta-repo/flathub-beta.flatpakrepo
+    $ flatpak install flathub-beta com.github.virtual_puppet_project.vpuppr
+```
+
+Maintainer: [Oro](https://github.com/orowith2os)
 
 ### Package manager
 
-Under construction.
+#### OpenSeeFace-GD (old)
+
+installation for [arch](https://aur.archlinux.org/packages/puppeteer)

```

Comment:
Sentence should start with a capital letter, "arch" should be "Arch Linux" since arch is a generic computer term that stands for "architecture" (e.g. x86, aarch)